strings.txt

```diff
@@ -26635,3 +26635,21 @@ SETUP_YEARMENU_1
 	NL	Toon jaartal van album en nummer
 	PT	Mostrar o ano do album e de cada pista
 
+SETUP_ALBUMLINKROLES
+	DE	Rollenlinks in Albumlisten anzeigen
+	EN	Show role links in album lists
+	ES	Mostrar enlaces de roles en listas de álbumes
+	FR	Afficher les liens de rôle dans les listes d'albums
+	HU	Szerepkörhivatkozások megjelenítése az albumlistákban
+	NL	Rolkoppelingen weergeven in albumlijsten
+	PT	Mostrar ligações de funções nas listas de álbuns
+
+SETUP_ALBUMLINKROLES_DESC
+	DE	Sie können auswählen, welche Rollen als Links in Albumlisten angezeigt werden sollen. Der Hauptbeitragende des Albums, der vom Musikscanner identifiziert wurde, wird immer angezeigt.
```

Comment:
"Sie können auswählen, welche Rollen als Links in Albumlisten angezeigt werden sollen. Der Hauptkünstler des Albums wird immer angezeigt"